src/components/box.jsx

```diff
@@ -0,0 +1,19 @@
+import PropTypes from "prop-types";
+
+function Box({title, value, className}) {
+  return (
+    <div className={`box ${className}`}>
+      <p>{title}</p>
+      <p>{value}</p>
+    </div>
+  )
+
+}
+
+export default Box;
+
+Box.propTypes = {

```

Comment:
propTypes is niet persé noodzakelijk omdat de data allemaal intern binnen de applicatie zit. Wanneer de data vanuit een API naar binnen wordt gehaald is het gebruik van propTypes wel interessant omdat je zo mogelijke problemen kan opvangen wanneer bijvoorbeeld de data die je uit de API haalt is veranderd.